spec/system/project_show_spec.rb

```diff
@@ -31,13 +31,16 @@
     context "Navigation Buttons" do 
       it "Shows the correct nav buttons for an approved project" do
         sign_in sponsor_user
+        sponsor_user["eligible_sponsor"] = true
```

Comment:
The factory for a sponsor user (on line 6) should be utilized in this test instead of setting the flag. 
```
let(:sponsor_user) { FactoryBot.create(:project_sponsor, uid: "pul123") }
```

Reply:
I agree with Carolyn. Again, the characteristics of a project sponsor are likely to change over time, so using the factory will keep our tests more robust as things inevitably shift.